merkletree/str.go

```diff
@@ -79,3 +79,15 @@ func (str *SignedTreeRoot) PreviousSTRHash() []byte {
 func (str *SignedTreeRoot) Signature() []byte {
 	return str.sig
 }
+
+func (str *SignedTreeRoot) LibVersion() []byte {
```

Comment:
These can change in epochs, thus I think it should be here, corresponding with policies of a specific STR. How does that sound?